share/zm-rpcapi.lsb

```diff
@@ -27,7 +27,7 @@ STARMAN=`PATH="$PATH:/usr/local/bin" /usr/bin/which starman`
 . /lib/lsb/init-functions
 
 start () {
-    $STARMAN --user=$USER --group=$GROUP --error-log=$LOGFILE --pid=$PIDFILE --listen=$LISTENIP:$LISTENPORT --preload-app --daemonize $BINDIR/zonemaster_backend_rpcapi.psgi || exit 1
+    $STARMAN --daemonize --preload-app --user=$USER --group=$GROUP --pid=$PIDFILE --error-log=$LOGFILE --listen=$LISTENIP:$LISTENPORT /usr/local/bin/zonemaster_backend_rpcapi.psgi || exit 1
```

Comment:
This removes the use of `$BINDIR`, is this on purpose? If so could you remove the variable definition?

Reply:
No, that was a mistake. Thanks!